doc/modules/clustering.rst

```diff
@@ -814,6 +825,91 @@ by black points below.
    In Proceedings of the 2nd International Conference on Knowledge Discovery
    and Data Mining, Portland, OR, AAAI Press, pp. 226–231. 1996
 
+.. _optics:
+
+OPTICS
+======
+
+The :class:`OPTICS` algorithm shares many similarities with the :class:`DBSCAN`
+algorithm, and can be considered a generalization of DBSCAN that relaxes the
+``eps`` requirement from a single value to a value range. The key difference
+between DBSCAN and OPTICS is that the OPTICS algorithm builds a *reachability*
+graph, which assigns each sample both a ``reachability_`` distance, and a spot
+within the cluster ``ordering_`` attribute; these two attributes are assigned
+when the model is fitted, and are used to determine cluster membership. If
+OPTICS is run with the default value of *inf* set for ``max_eps``, then DBSCAN
+style cluster extraction can be performed in linear time for any given ``eps``
+value using the ``cluster_optics_dbscan`` method. Setting ``max_eps`` to a
+lower value will result in shorter run times, and can be thought of as the
+maximum cluster object size (in diameter) that OPTICS will be able to extract.
```

Comment:
No. "Maximum cluster object size (in diameter)" clearly is wrong. Cluster are not bound in _diameter_ by max_eps at all.

Reply:
true, changed to "the maximum neighborhood radius from each point to find other potential reachable points."